External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -145,6 +145,7 @@ void OpDispatchBuilder::SyscallOp(OpcodeArgs) {
   if (Op->TableInfo->Flags & X86Tables::InstFlags::FLAGS_BLOCK_END) {
     // RIP could have been updated after coming back from the Syscall.
     NewRIP = _LoadContext(GPRSize, GPRClass, offsetof(FEXCore::Core::CPUState, rip));
+    CalculateDeferredFlags();
```

Comment:
Are all the added `CalculateDeferredFlags` calls in this file necessary? I've noticed a few end up being redundant. Which would be fine but I want to make sure that they weren't thrown around during debugging and no longer necessary.

Reply:
Probably some could be removed, but the intention is to call `CalculateDeferredFlags` on any block boundary, since if you try to preserve `CachedNZCV` across block boundaries you're in for extreme badness. Calling `CalculateDeferredFlags` before things like function ends makes all those tricky bugs go away. The no-op calls should be cheap so I'm inclined to leave them in. And someday refactor this mess to make "add a new block" properly factored.